apps/hubble/src/storage/stores/storeEventHandler.ts

```diff
@@ -52,8 +52,12 @@ const STORE_TO_SET: Record<StoreType, UserMessagePostfix> = {
   [StoreType.USER_DATA]: UserPostfix.UserDataMessage,
   [StoreType.VERIFICATIONS]: UserPostfix.VerificationMessage,
   [StoreType.USERNAME_PROOFS]: UserPostfix.UsernameProofMessage,
+  [StoreType.TAGS]: UserPostfix.TagMessage,
+  [StoreType.OBJECTS]: UserPostfix.ObjectMessage,
+  [StoreType.RELATIONSHIPS]: UserPostfix.RelationshipMessage,
 };
 
+// VLAD-TODO: are our new msg types "prunable"?
```

Comment:
Nice!
We will need to talk about storage on H2 at some point.
- How will it be managed (by L2 contracts?)
- Should H2 msgs even be pruned?